src/libraries/System.IO.FileSystem/tests/File/OpenHandle.cs

```diff
@@ -7,28 +7,24 @@
 namespace System.IO.Tests
 {
     // to avoid a lot of code duplication, we reuse FileStream tests
-    public class File_OpenHandle : FileStream_ctor_options_as
+    public class File_OpenHandle : FileStream_ctor_options
     {
         protected override string GetExpectedParamName(string paramName) => paramName;
 
         protected override FileStream CreateFileStream(string path, FileMode mode)
         {
             FileAccess access = mode == FileMode.Append ? FileAccess.Write : FileAccess.ReadWrite;
-            return new FileStream(File.OpenHandle(path, mode, access, preallocationSize: PreallocationSize), access);
+            return new FileStream(File.OpenHandle(path, mode, access), access);
```

Comment:
What was the verdict on testing coverage with all of these tests being updated to not pass in a preallocation size?

Reply:
Coverage is good now.

I refactored tests to follow the pattern that was already used by adding an overload of `CreateFileStream` to the base class `FileStream_ctor_options` that accepts a `preallocationSize` and add the `preallocationSize` related tests in this class. The derived classes use it from different public APIs.